src/univariate/continuous/generalizedpareto.jl

```diff
@@ -15,8 +15,9 @@ f(x; \\mu, \\sigma, \\xi) = \\begin{cases}
 ```
 
 ```julia
-GeneralizedPareto()             # Generalized Pareto distribution with unit shape and unit scale, i.e. GeneralizedPareto(0, 1, 1)
-GeneralizedPareto(μ, σ)         # Generalized Pareto distribution with shape ξ and scale σ, i.e. GeneralizedPareto(0, σ, ξ)
+GeneralizedPareto()             # Standard exponential distribution (GPD with shape 0, location 0, scale 1)
+GeneralizedPareto(ξ)            # Generalized Pareto distribution with shape ξ, location 0, and scale 1
```

Comment:
This constructor is not defined.

I think it would be good to follow the existing examples and show the corresponding 3-argument constructor as well:
```suggestion
GeneralizedPareto(ξ)            # Generalized Pareto distribution with shape ξ and unit scale, i.e. GeneralizedPareto(0, 1, ξ)
```